woocommerce/woocommerce
# Added Missing Blocks Linting Change Trigger

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This adds the `tests` directory to `@woocommerce/block-library`'s `config.ci` entries. There are a lot of JS files inside that are linted but wouldn't trigger the lint job. [This PR](https://github.com/woocommerce/woocommerce/pull/44228) solved one such issue caused by the linting job not running.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new branch based off this one. In my case I made `fix/blocks-linting-ci-test`.
2. `pnpm utils ci-jobs --base-ref='fix/blocks-linting-ci'` should return nothing.
3. Make a change somewhere in `plugins/woocommerce-blocks/tests` to some JS file.
4. `pnpm utils ci-jobs --base-ref='fix/blocks-linting-ci'` again should return a lint job.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

CI-only change.

</details>
